examples/components/RadarDemo.vue

```diff
@@ -42,39 +39,6 @@ export default {
   components: {
     'v-radar': Radar
   },
-  methods: {
-    handleDragstart(component) {
-      const shape = component.getStage();
-      const dragLayer = vm.$refs.dragLayer.getStage();
-      const stage = vm.$refs.stage.getStage();
-
-      // moving to another layer will improve dragging performance
-      shape.moveTo(dragLayer);
-      stage.draw();
-
-      component.config.shadowOffsetX = 15;
-      component.config.shadowOffsetY = 15;
-      component.config.scaleX = component.config.startScale * 1.2;
-      component.config.scaleY = component.config.startScale * 1.2;
-    },
-    handleDragend(component) {
-      const shape = component.getStage();
-      const layer = vm.$refs.layer.getStage();
-      const stage = vm.$refs.stage.getStage();
-
-      shape.moveTo(layer);
-      stage.draw();
-
-      shape.to({
-        duration: 0.5,
-        easing: Konva.Easings.ElasticEaseOut,
-        scaleX: component.config.startScale,
-        scaleY: component.config.startScale,
-        shadowOffsetX: 5,
-        shadowOffsetY: 5
-      });
-    }
-  },
```

Comment:
Ok so you started to refactor dragstart and dragend, forgot about my previous comment in previous commit 😄